content-scope-scripts/content-scope-scripts-impl/src/main/java/com/duckduckgo/contentscopescripts/impl/messaging/ContentScopeScriptsWebMessagingPlugin.kt

```diff
@@ -89,56 +88,68 @@ class ContentScopeScriptsWebMessagingPlugin @Inject constructor(
                     }
 
                     // Process global handlers first (always processed regardless of feature handlers)
-                    globalHandlers.getPlugins()
+                    globalHandlers
+                        .getPlugins()
                         .map { it.getGlobalJsMessageHandler() }
                         .filter { it.method == jsMessage.method }
                         .forEach { handler ->
                             handler.process(jsMessage)?.let { processResult ->
                                 when (processResult) {
                                     is SendToConsumer -> {
-                                        sendToConsumer(jsMessageCallback, jsMessage, replyProxy)
+                                        sendToConsumer(webView, jsMessageCallback, jsMessage, replyProxy)
                                     }
                                     is SendResponse -> {
-                                        onResponse(jsMessage, replyProxy)
+                                        appCoroutineScope.launch {
```

Comment:
we are trying to get away from `appCoroutineScope.launch` in these flows; is this required?